ui/src/redux/thunks/collections.js

```diff
@@ -328,7 +328,7 @@ export const updateTags =
             dispatch(
                 updateTagsToStore(
                     getFullCollectionName(domain, collectionName, category),
-                    detail.tags,
+                    detail,
```

Comment:
can you help me understand this change?

Reply:
Previously, we had a conditional check for handling role and group tags in the redux reducers files. If the tags of a collection were empty, we used to create a collection(role\group) with only the tags as an attribute. However, I made a decision to change this approach. Now, regardless of whether the tags are empty or not, we will always create the collection and add it to the store so we need to send the collection data and not only its tags.